src/lib/DriverMessage.mli

```diff
@@ -0,0 +1,15 @@
+(* Driver Messages come in 2 types, either an error message or a normalized term/Definition*)
```

Comment:
I like the new design of this module!

Reply:
Thanks! Yeah I was misunderstanding where the Syntax.pp was being used so I avoided adding more stuff to data. Now that I have a better handle on it, the latest patch should be much cleaner.